source/technical/reservations.rst

```diff
@@ -217,6 +217,27 @@ To change the number of nodes of a lease, click on the *Update Lease* button in
 Navigate to the "Hosts" tab,  and fill out the form by specifying the new minimum 
 and maximum numbers of hosts. Then, click on the *Update* button to finish your request.
 
+Changing the Number of Floating IPs in a Lease
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+It is possible to change the number of floating IPs in a lease, whether the
+lease is pending or active. In some situations, you cannot renew a lease due to
+another user reserving the same floating IP in your lease. In this case, you
+can set your lease to have 0 floating IPs, and instead use an
```

Comment:
Or maybe make a new separate lease for a different floating IP? I think over time it would be nice if the UX for reserving IPs was good enough that we could stop allowing the ad-hoc ones.